Scripts/script-StixCreator.yml

```diff
@@ -0,0 +1,118 @@
+commonfields:
+  id: StixCreator
+  version: -1
+name: StixCreator
+script: |-
+  ''' IMPORTS '''
+  import json
+
+  from stix2 import Indicator, Bundle
+
+  indicators = []
+
+  userArgs = demisto.args().get('indicators','Unknown')
+  if isinstance(userArgs, dict) == True:
+      all_args = json.loads(json.dumps(userArgs))
+  else:
+      all_args = json.loads(demisto.args().get('indicators','Unknown'))
+
+  counter = 0
+  for indicator_fields in all_args:
```

Comment:
if all_args is list then why you check that it is dict in line 14?

Reply:
because currently there are 2 ways to use this script one is from the server (which send golang dictionary) and one is from the user interface (! - the bang), so when the indicators are being sent from the user it doesn't get as dict, but as string